binary_gentoo/internal/cli/build.py

```diff
@@ -125,6 +126,11 @@ def parse_command_line(argv):
                         action='store_true',
                         help='enforce installation (default: build but do not install)')
 
+    parser.add_argument('--tag-docker-image',
+                        metavar='IMAGE',
+                        dest='tag_docker_image',
+                        help='create a Docker image of the resulting container')
```

Comment:
English is not my native language, but preposition "of" would better be "from" (or "off" with two "f") in my mind. What do you think?

Reply:
Yes, that does sound good.